styleguide/derek/_partials/ticker.ejs

```diff
@@ -0,0 +1,12 @@
+<section class="ticker">
+  <section class="container">
+    <div class="row">
```

Comment:
Didn't you create a `make-container()` mixin? Then we could have a ticker-container class instead of standard container class for maintainable css purposes. 

Also, use ticker-row instead of row and then do `make-row();`